src/redux/missions/missions.js

```diff
@@ -0,0 +1,43 @@
+import { createAsyncThunk, createSlice, current } from '@reduxjs/toolkit';
+import getMissionsApi from '../../api/SpaceApi';
+
+const LOAD_MISSIONS = 'LOAD_MISSIONS';
+
+// LOAD
+export const loadMissions = createAsyncThunk(LOAD_MISSIONS, async () => {
+  try {
+    const response = await getMissionsApi();
+    return response.data.map((mission) => ({ ...mission, reserved: false }));
```

Comment:
- [ ] Kindly make sure to store only selected data in the redux store. I highly recommend you store the only needed data.
 
_**Suggestion**_
`return response.data.map((mission) => ({
      id: mission.mission_id,
      name: mission.mission_name,
      description: mission.description,
      reserved: false,
    }))`